CMakeLists.txt

```diff
@@ -45,6 +45,19 @@ ELSE()
       OFF
   )
 
+  # Define option for mixed precision
```

Comment:
I 100% agree with this belonging in MPACT, not Futility.  Futility as a base library provides `SRK`, `SSK`, and `SDK` definitions.  How those definitions get used is MPACT-specific (or whatever code).  I think MPACTUtils is the correct place for now.

Reply:
I think the mixed precision is pretty general concept not tied to MPACT, although we first implemented it in MPACT. The reason to have it in Futility is we need this to control the behavior of several operators. I see the questioning of adding the operators and I will explain in the following comments.

Reply:
The concept is general, but the implementation is not.  What I'm more concerned about is that if MPACT wants it on, but MAMBA wants it off.  If its implemented in Futility, we don't have any options to turn it on and off between the codes.  Like I mentioned below, I think having mixed approxeq routines defined here, but I don't think defining the parameter here makes sense because each code will want to treat this differently (and nothing in Futility is doing mixed precision at all).

Reply:
Sounds good to move the parameters.